.github/workflows/dart_ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         if: always() && steps.install.outcome == 'success'
 
       - name: Analyze project source
-        run: dart run dart_dev analyze
+        run: dart analyze
```

Comment:
switched to dart analyze since dart_dev still runs dartanalyzer and that fails on newer Dart versions